pkg/cloudprovider/providers/vsphere/vsphere.go

```diff
@@ -100,6 +100,23 @@ type VSphereConfig struct {
 		VMName string `gcfg:"vm-name"`
 	}
 
+	VirtualCenter map[string]*struct {
+		// vCenter username.
+		User string `gcfg:"user"`
+		// vCenter password in clear text.
+		Password string `gcfg:"password"`
+		// vCenter port.
+		VCenterPort string `gcfg:"port"`
+		// True if vCenter uses self-signed cert.
+		InsecureFlag bool `gcfg:"insecure-flag"`
+		// Datacenter in which VMs are located.
+		Datacenter string `gcfg:"datacenter"`
```

Comment:
Also how to identify which VirtualCenter map entry is master? I think we should add a field to specify which of the VC entry is a master? Otherwise we would not know it.